### PR TITLE
Features page responsive

### PR DESCRIPTION
**Pull request by koksyncho**
features.html:
-added an encapsulating div with class "features" which spans across the whole body so that we can make the difference between the index.html and the features.html in our style.css file (this is why the changes which github shows are so messy);
-split the col-8 which has the tesplan.png image in it into col-3, col-2 and col-3 classes for better styling;
-removed some of the elements from the html and added them from the css (mainly the last check-list with the display flex property); 

style.css:
-changes to the width and alignment of the different col-? classes and the img and li tags;

### CODE REVIEW

File: content/pages/features.html [hunk 1]
```diff
@@ -5,152 +5,157 @@
         <meta name="url" content="features/">
     </head>
     <body>
-        <!-- dashboard -->
-        <section class="col-12 secondary">
-            <div class="col-12 screenshot-title">
-                <h3>A central dashboard to start your day</h3>
-            </div>
+        <div class="features">
```
Comment (atodorov) [inline content/pages/features.html]: You can break the indenting rules a little bit here and indent with 2 spaces from the body tag and not indent the rest of the blocks. Let me see if I can review the rest easily as-is, if not you can apply the change I'm talking about.

Comment (atodorov) [reply content/pages/features.html]: Hmmm, I have a better idea, why not apply the features class to the body tag instead ? If you're using it as a selector only then this will do.

File: content/pages/features.html [hunk 2]
```diff
@@ -5,152 +5,157 @@
...
+                                <i class="fa fa-check"></i>
+                                <strong>Execute:</strong> create Test Run for each build and environment
+                                that will be tested to record the results.
+                            </li>
+                        </ul>
                     </div>
+                    
+                    <div class="col-2"></div>
+
+                    <div style="margin-bottom: 5rem;" class="col-3">
```
Comment (atodorov) [inline content/pages/features.html]: Compare how the current layout looks vs. what's on the live website and notice that the image goes a bit to the right on a regular desktop screen. This is because of the style nesting you have introduced. The col-3 class here means 25% of the encapsulating col-8 above (which is 66,66%). This is not 25% of the overall page width!

I think you should work with the previous styles, but to make things easier to spot you will have to apply the body/features change first so the diff is easier to review.

File: content/pages/features.html [hunk 3]
```diff
@@ -5,152 +5,157 @@
...
+                            </li>
+
+                            <li>
+                                <i class="fa fa-check"></i>
+                                <strong>Execute:</strong> create Test Run for each build and environment
+                                that will be tested to record the results.
+                            </li>
+                        </ul>
                     </div>
+
```
Comment (atodorov) [inline content/pages/features.html]: Line with spaces here, remove the spaces.

**Review by atodorov — CHANGES_REQUESTED** [review]
Requested changes to the HTML code.

Responsive layout look good overall. The two bulleted lists need to have the same padding as the rest of the elements, especially on the left-hand side to create an interrupted visual line.

The action buttons need some vertical spacing between them when they appear stacked on top of each other.

Note. I don't see you hiding the empty col-2 divs, which are essentially used to center the content, on smaller screens. You can try display:none; width:0 for them and they will free up some space for you.

**Comment by atodorov** [comment]
Also use 'git rebase' instead of merge and drop the merge commit!

**Comment by koksyncho** [comment]
Thanks for the feedback! Dropping the merge commit now.

**Comment by koksyncho** [comment]
Integrated most of the requested changes. The ones which haven't been done accordingly are:
-the "features" class is still in the encapsulating div, the spacing was done with 2 spaces instead of the previous 4 (had troubles with linking the class to the body);
-tried to match the layout of that one section (the one with the tesplan.png) to the live version while still using the col-separation, it is similar but not entirely;

File: theme/static/style/style.css [hunk 4]
```diff
@@ -403,6 +408,33 @@ pre code {
...
+        max-width: 95%;
+        float: none;
+        margin: auto;
+        display: block;
+    }
+    .features li {
+        padding-left: 2.5%;
+        padding-right: 2.5%;
+    }
+    .features .space-filler {
```
Comment (atodorov) [inline theme/static/style/style.css]: `.features .col-2` and get rid of space-filler.

File: theme/static/style/style.css [hunk 5]
```diff
@@ -450,4 +482,44 @@ pre code {
...
+        display: block;
+    }
+    .features li {
+        padding-left: 2.5%;
+        padding-right: 2.5%;
+    }
+    .features .demo-btn {
+        margin-bottom: 2.5%;
+    }
+    .features .space-filler {
```
Comment (atodorov) [inline theme/static/style/style.css]: .features .col-2

File: content/pages/features.html [hunk 6]
```diff
@@ -136,21 +140,22 @@ <h3>Integrate with your favorite bug tracker</h3>
...
-            <div class="col-2"></div>
+            <div class="col-2 space-filler"></div>
 
             <div class="col-8" style="padding-top: 2rem; padding-bottom: 2rem;">
-                <div>
-                    <a href="https://demo.kiwitcms.org" class="action-btn accent-important">Checkout Demo</a>
-                    <a href="http://kiwitcms.readthedocs.io" class="action-btn accent-unimportant">Read Documentation</a>
+                <div style="text-align: center;">
+                    <a href="https://demo.kiwitcms.org" class="action-btn demo-btn accent-important">Checkout Demo</a>
+                    <a href="http://kiwitcms.readthedocs.io" class="action-btn documentation-btn accent-unimportant">Read Documentation</a>
```
Comment (atodorov) [inline content/pages/features.html]: documentation-btn isn't used anywhere so remove it.

File: theme/static/style/style.css [hunk 7]
```diff
@@ -359,10 +359,15 @@ pre code {
     font-size: 3.5rem;
 }
 
+/* Features Page */
+
+.features .horizontally-stacked-list {
```
Comment (atodorov) [inline theme/static/style/style.css]: Simply .horizontally-stacked-list is enough and remove the comment above!

File: theme/static/style/style.css [hunk 8]
```diff
@@ -403,6 +408,33 @@ pre code {
         text-align: left;
     }
 
+    /*features section*/
```
Comment (atodorov) [inline theme/static/style/style.css]: features page, not section

File: theme/static/style/style.css [hunk 9]
```diff
@@ -450,4 +482,44 @@ pre code {
...
     }
+
+    /*features section*/
+    .features .w50 {
+        padding-top: 2rem;
+        min-width: 95%;
+        max-width: 95%;
+    }
+    .features .col-8 {
+        min-width: 100%;
```
Comment (atodorov) [inline theme/static/style/style.css]: I think for all of these properties max-width is enough.

File: theme/static/style/style.css [hunk 10]
```diff
@@ -450,4 +482,44 @@ pre code {
...
+    }
+    .features img {
+        min-width: 95%;
+        max-width: 95%;
+        float: none;
+        margin: auto;
+        display: block;
+    }
+    .features li {
+        padding-left: 2.5%;
```
Comment (atodorov) [inline theme/static/style/style.css]: IMO it is better to use 1.8rem as we do everywhere else. You can do the same for img as well, set max-width to 100% and update the border-size property to border-box to make sure the 100% contain all padding. margin and borders.

**Review by atodorov — CHANGES_REQUESTED** [review]
Other changes requested, still there's a merge commit which you need to get rid of. Also squash all of the rest commits when you rebase.

**Comment by atodorov** [comment]
> (had troubles with linking the class to the body)

The <body> tag inside the features.html file is not the <body> tag that appears on the rendered features page! This is simply a holder for the content block of that page (e.g. like begin-end). When the content is rendered Pelican takes the structure from base.html template and replaces with the content and other blocks. 

The last commit on the source branch introduces the ability to configure a body_class variable on the page which gets used in the template. When you rebase you can remove the wrapping div and your CSS selectors should work like they do ATM.

File: content/pages/features.html [hunk 11]
```diff
@@ -20,7 +20,7 @@ <h3>A central dashboard to start your day</h3>
                 </div>
 
                 <div class="w50" style="float: left;">
-                    <div class="about-panel-title">For test engineers</div>
+                    <div class="about-panel-title">For test enginners</div>
```
Comment (atodorov) [inline content/pages/features.html]: Somehow the rebase didn't work for you but the latest source branch has this typo fixed.

File: theme/static/style/style.css [hunk 12]
```diff
@@ -248,8 +248,7 @@ pre code {
 
 /* Screenshots Section */
 
-/* Title */
-.screenshot-title {
+.screenshot-title{
```
Comment (atodorov) [inline theme/static/style/style.css]: Don't do this, we prefer having a space before {, it's that some of the blocks are missing it, but this PR purpose is not the fix the styles we don't care about.

File: theme/static/style/style.css [hunk 13]
```diff
@@ -263,7 +262,7 @@ pre code {
     margin-top: 1.5rem;
 }
 
-.img {
+.img{
```
Comment (atodorov) [inline theme/static/style/style.css]: don't remove space

File: theme/static/style/style.css [hunk 14]
```diff
@@ -292,8 +291,7 @@ pre code {
     overflow: hidden;
 }
 
-/* Image Count */
-.img-index {
+.img-index{
```
Comment (atodorov) [inline theme/static/style/style.css]: don't remove space

File: theme/static/style/style.css [hunk 15]
```diff
@@ -405,6 +406,33 @@ pre code {
         text-align: left;
     }
 
+    /*features page*/
+    .w50 {
```
Comment (atodorov) [inline theme/static/style/style.css]: prefix with .features

File: theme/static/style/style.css [hunk 16]
```diff
@@ -405,6 +406,33 @@ pre code {
...
+        max-width: 95%;
+        float: none;
+        margin: auto;
+        display: block;
+    }
+    .features li {
+        padding-left: 1.8rem;
+        padding-right: 1.8rem;
+    }
+    .features .space-filler {
```
Comment (atodorov) [inline theme/static/style/style.css]: No such class used in the page so remove the style definition.

File: theme/static/style/style.css [hunk 17]
```diff
@@ -452,15 +480,19 @@ pre code {
...
-
-
-/*
-    Hiding the "Loading" text that appears
-    on the bottom of the screen, caused by
-    the jQuery Mobile scripts.
-*/
 
-.ui-loader {
-    display: none;
```
Comment (atodorov) [inline theme/static/style/style.css]: Don't remove this last definition, we still need it.

**Review by atodorov — CHANGES_REQUESTED** [review]

**Comment by atodorov** [comment]
@ivo0126 please provide a review for this PR

File: content/pages/features.html [hunk 18]
```diff
@@ -49,7 +49,7 @@ <h3>A Test Plan is a collection of Test Cases</h3>
             <div class="col-2"></div>
 
             <div class="col-8">
-                <div style="float:left" class="w25">
+                <div style="float:left" class="col-3 w25">
```
Comment (ivo0126) [inline content/pages/features.html]: Move styles to `style.css`.
same goes for lines:
`22, 31, 52, 76, 116, 138, 150, 151`.

**Review by ivo0126 — CHANGES_REQUESTED** [review]

File: theme/static/style/style.css [hunk 19]
```diff
@@ -1,6 +1,6 @@
 /* Reset */
 
-*{
+* {
```
[the remark this answers is not in the document]
Comment (atodorov) [reply theme/static/style/style.css]: I agree with you that these changes should be in a separate commit, but let's leave it like this this time.

File: content/pages/features.html [hunk 20]
```diff
@@ -71,9 +71,12 @@ <h3>A Test Plan is a collection of Test Cases</h3>
                     </ul>
                 </div>
 
-                <div style="float: right; clear:right;margin-bottom: 5rem;">
+                <div class="col-2"></div>
+
+                <div style="margin-bottom: 5rem;" class="col-3">
```
Comment (atodorov) [inline content/pages/features.html]: Revert this change because it skews the layout of the image and follow @ivo0126 advice to move these style definitions as CSS classes.

File: theme/static/style/style.css [hunk 21]
```diff
@@ -374,6 +372,12 @@ pre code {
     }
 }
 
+/* Features Page */
+
+.horizontally-stacked-list {
```
Comment (atodorov) [inline theme/static/style/style.css]: Move this block above the media selector for width 1360px.

File: theme/static/style/style.css [hunk 22]
```diff
@@ -440,6 +444,34 @@ pre code {
...
+        display: block;
+    }
+    .features li {
+        padding-left: 1.8rem;
+        padding-right: 1.8rem;
+    }
+    .features .col-2 {
+        display: none; 
+        width: 0;
+    }
```
Comment (atodorov) [inline theme/static/style/style.css]: Looking at the above block I think most of it needs to go into the media selector for width 1360px so we can display images nicely even in landscape mode.

File: theme/static/style/style.css [hunk 23]
```diff
@@ -463,8 +495,22 @@ pre code {
         padding-left: 1.8rem;
         box-sizing: border-box;
     }
-}
 
+    /*features page*/
+    .features .w50 {
+        min-width: 95%;
+        max-width: 95%;
```
Comment (atodorov) [inline theme/static/style/style.css]: I think something like width: 100% and padding (or margin) 1.8rem on both sides, probably combined with box-sizing: border-box will have the same resizing effect and allow all the elements to have the same alignment.

**Review by atodorov — CHANGES_REQUESTED** [review]
To recap:
* keep the white space changes as they are, but in the future don't do this
* Revert the changes around the new col-3 class
* adjust class definitions override for tablet screens (or smaller PC screen) and work from there.

**Comment by koksyncho** [comment]
Thanks for the reviews! I think that I followed your instructions, but upon committing the changes, not all of the requests disappeared (I may have screwed something up - I used "git reset HEAD~1" to cancel my first commit).

File: theme/static/style/style.css [hunk 24]
```diff
@@ -325,43 +323,93 @@ pre code {
...
 }
 
+/* Features Page */
+
+.horizontally-stacked-list {
+    display: flex;
+    float: left;
+}
+
+.w50 {
```
Comment (atodorov) [inline theme/static/style/style.css]: Don't redefine the .w50 class here. This is causing its definition to be in 2 places. Better add something like .float-left class and apply it to the containers you want floating on the left.

File: theme/static/style/style.css [hunk 25]
```diff
@@ -325,43 +323,93 @@ pre code {
...
+    margin-bottom: 5rem;
+}
+
+.call-to-action-field {
+    padding-top: 2rem;
+    padding-bottom: 2rem;
+}
+
+.call-to-action-buttons {
+    text-align: center;
```
Comment (atodorov) [inline theme/static/style/style.css]: Can this text-align go into the .call-to-action-field definition? If yes then you can remove this additional class.

File: theme/static/style/style.css [hunk 26]
```diff
@@ -388,15 +436,11 @@ pre code {
 
 
     /* Screenshots section */
-    .screenshots .col-2, .img-index {
+    .screenshots .img-index {
```
Comment (atodorov) [inline theme/static/style/style.css]: Revert this change, not related to current feature.

File: theme/static/style/style.css [hunk 27]
```diff
@@ -388,15 +436,11 @@ pre code {
...
         display: none;
         width: 0;
     }
 
-    .screenshots .col-8 {
-        width: 100%;
-    }
-
```
Comment (atodorov) [inline theme/static/style/style.css]: revert this, unrelated to current feature

File: theme/static/style/style.css [hunk 28]
```diff
@@ -432,6 +476,39 @@ pre code {
...
+        max-width: 95%;
+        float: none;
+        margin: auto;
+        display: block;
+    }
+    .features li {
+        padding-left: 1.8rem;
+        padding-right: 1.8rem;
+    }
+    .w25 {
```
Comment (atodorov) [inline theme/static/style/style.css]: should be `.features .w25`

File: theme/static/style/style.css [hunk 29]
```diff
@@ -432,6 +476,39 @@ pre code {
...
+        min-width: 100%;
+        max-width: 100%;
+    }
+    .testplan {
```
Comment (atodorov) [inline theme/static/style/style.css]: Prefix with `.features` just for consistency with the rest and for better clarity.

File: theme/static/style/style.css [hunk 30]
```diff
@@ -455,8 +532,24 @@ pre code {
...
-}
 
+    /*features page*/
+    .features .w50 {
+        width: 100%;
+        padding-left: 1.8rem;
+        padding-right: 1.8rem;
+        box-sizing: border-box;
+    }
+    .col-8 .horizontally-stacked-list {
```
Comment (atodorov) [inline theme/static/style/style.css]: Prefix with `.features` for consistency and clarity.

File: theme/static/style/style.css [hunk 31]
```diff
@@ -325,43 +323,93 @@ pre code {
...
+.call-to-action-field {
+    padding-top: 2rem;
+    padding-bottom: 2rem;
+}
+
+.call-to-action-buttons {
+    text-align: center;
+}
+
+@media screen and (max-width: 1360px){
```
Comment (atodorov) [inline theme/static/style/style.css]: This looks like it should not be here. Probably result of conflicts.

**Review by atodorov — CHANGES_REQUESTED** [review]
Please rebase to latest and resolve conflicts. Then fix all the requested changes.

File: content/pages/features.html [hunk 32]
```diff
@@ -71,9 +71,10 @@ <h3>A Test Plan is a collection of Test Cases</h3>
                     </ul>
                 </div>
 
-                <div style="float: right; clear:right;margin-bottom: 5rem;">
-                    <img src="/images/features/testplan.png" class="margin0 shadow">
+                <div class="testplan">
```
Comment (atodorov) [inline content/pages/features.html]: There's no definition for the `.testplan` class with the removed properties.

File: content/pages/features.html [hunk 33]
```diff
@@ -144,9 +145,9 @@ <h3>Integrate with your favorite bug tracker</h3>
         <section class="col-12 secondary">
             <div class="col-2"></div>
 
-            <div class="col-8" style="padding-top: 2rem; padding-bottom: 2rem;">
-                <div>
-                    <a href="https://demo.kiwitcms.org" class="action-btn accent-important">Checkout Demo</a>
+            <div class="col-8 call-to-action-field">
+                <div class="call-to-action-buttons">
```
Comment (atodorov) [inline content/pages/features.html]: call-to-action-buttons is not used, remove it.

**Review by atodorov — CHANGES_REQUESTED** [review]
Please make the requested changes and also resolve the conflicts so we can proceed further.

File: content/pages/features.html (hunk 32 above)
Comment (ivo0126) [reply content/pages/features.html]: What do You mean by that?

The styles for `.testplan` are defined on line `502`, if that's what You're talking about.

Or are You saying that You want the properties `float: right` and `clear: right` to be returned, since they are no longer used?

Comment (atodorov) [reply content/pages/features.html]: Yes, I'm saying the float and clear properties need to come back. Without them this change breaks the existing layout when on a large screen.

Comment (koksyncho) [reply content/pages/features.html]: Thanks for the clarification, would not have understood it otherwise.

File: content/pages/features.html [hunk 34]
```diff
@@ -110,8 +111,8 @@ <h3>Integrate with your favorite bug tracker</h3>
             <div class="col-2"></div>
 
             <div class="col-8">
-                <div style="float:left">
-                    <ul class="check-list" style="display: flex">
+                <div>
```
Comment (ivo0126) [inline content/pages/features.html]: Delete this `div`, since it does nothing.

File: theme/static/style/style.css [hunk 35]
```diff
@@ -325,52 +323,90 @@ pre code {
...
+    padding-top: 2rem;
+    padding-bottom: 2rem;
+    text-align: center;
+}
+
+.about-panel {
+    padding-top: 2rem;
+}
+
 @media screen and (max-width: 1360px){
```
Comment (ivo0126) [inline theme/static/style/style.css]: When the window width is shrinked enough for the styles in this query to apply, the images go off-center and stick to the left side of the screen, which does not fit with the overall style of the page. This can be fixed by adding the following properties inside of the query:

```
.col-8 {
    width: 97%;
}

.margin0 {
    margin-left: 7.5%;
    width: 87%;
}
```

`.margin0` refers to the images.

Or try finding another solution if this one doesn't suit you.

File: theme/static/style/style.css [hunk 36]
```diff
@@ -400,11 +436,15 @@ pre code {
 
 
     /* Screenshots section */
-    .screenshots .img-index {
+    .screenshots .col-2 .img-index {
```
Comment (ivo0126) [inline theme/static/style/style.css]: There  has to be a comma between `.col-2` and `.img-index {`, so the whole thing will look like `.screenshots .col-2, .img-index {`, as demanded by @atodorov.

File: content/pages/features.html (hunk 34 above)
Comment (ivo0126) [inline content/pages/features.html]: Delete this `div`, since it does nothing.

File: theme/static/style/style.css (hunk 35 above)
Comment (ivo0126) [inline theme/static/style/style.css:395]: When the window width is shrinked enough for the styles in this query to apply, the images go off-center and stick to the left side of the screen, which does not fit with the overall style of the page. This can be fixed by adding the following properties inside of the query:

```
.col-8 {
    width: 97%;
}

.margin0 {
    margin-left: 7.5%;
    width: 87%;
}
```

`.margin0` refers to the images.

Or try finding another solution if this one doesn't suit you.

File: theme/static/style/style.css (hunk 36 above)
Comment (ivo0126) [inline theme/static/style/style.css]: There  has to be a comma between `.col-2` and `.img-index {`, so the whole thing will look like `.screenshots .col-2, .img-index {`, as demanded by @atodorov.

**Review by ivo0126 — CHANGES_REQUESTED** [review]

File: content/pages/features.html [hunk 37]
```diff
@@ -71,9 +71,10 @@ <h3>A Test Plan is a collection of Test Cases</h3>
                     </ul>
                 </div>
 
-                <div style="float: right; clear:right;margin-bottom: 5rem;">
-                    <img src="/images/features/testplan.png" class="margin0 shadow">
+                <div class="testplan">
+                    <img src="/images/features/testplan.png" class="testplan-img margin0 shadow">
                 </div>
+
```
Comment (atodorov) [inline content/pages/features.html]: Remove new line, not needed.

File: content/pages/features.html [hunk 38]
```diff
@@ -110,29 +111,28 @@ <h3>Integrate with your favorite bug tracker</h3>
...
+                    </li>
+
+                    <li>
+                        <i class="fa fa-github"></i>
+                        <strong>Integration:</strong> with GitHub, JIRA and Bugzilla.
+                        Others can be added very easily.
+                    </li>
+                </ul>
+
+
```
Comment (atodorov) [inline content/pages/features.html]: Remove 1 of the 2 new lines.

File: content/pages/features.html [hunk 39]
```diff
@@ -71,9 +71,10 @@ <h3>A Test Plan is a collection of Test Cases</h3>
                     </ul>
                 </div>
 
-                <div style="float: right; clear:right;margin-bottom: 5rem;">
-                    <img src="/images/features/testplan.png" class="margin0 shadow">
+                <div class="testplan">
+                    <img src="/images/features/testplan.png" class="testplan-img margin0 shadow">
```
Comment (atodorov) [inline content/pages/features.html]: Remove testplan-img class. It adds a double 5rem margin at the botton. The wrapping div already provides this.

File: theme/static/style/style.css [hunk 40]
```diff
@@ -325,52 +323,97 @@ pre code {
...
+    .features .col-2 {
+        display: none;
+        width: 0;
+    }
+    .features .col-8 {
+        width: 97%;
+    }
+    .features .margin0 {
+        margin-left: 7.5%;
+        width: 87%;
```
Comment (atodorov) [inline theme/static/style/style.css]: Use wid/max-width 100% and margin in rems, 1.8 is the default everywhere else I believe. You can also use box-sizing: border-box to make everything fit inside the container.

File: theme/static/style/style.css [hunk 41]
```diff
@@ -440,6 +487,39 @@ pre code {
...
+        min-width: 100%;
+        max-width: 100%;
+    }
+    .features .col-3 {
+        min-width: 100%;
+        max-width: 100%;
+    }
+    .features img {
+        min-width: 95%;
+        max-width: 95%;
```
Comment (atodorov) [inline theme/static/style/style.css]: Again, use width 100% and add padding and border-box properties to control spacing between elements. I think it is best if you comment out all styles for responsive and start working on each element one by one until you are satisfied with it.

File: theme/static/style/style.css [hunk 42]
```diff
@@ -440,6 +487,39 @@ pre code {
...
+        padding-right: 1.8rem;
+    }
+    .features .w25 {
+        min-width: 100%;
+        max-width: 100%;
+    }
+    .features .testplan {
+        min-width: 100%;
+        max-width: 100%;
+        margin-bottom: 5rem;
```
Comment (atodorov) [inline theme/static/style/style.css]: I think margin-bottom here is redundant, isn't it ?

**Review by atodorov — CHANGES_REQUESTED** [review]

File: theme/static/style/style.css (hunk 42 above)
Comment (koksyncho) [reply theme/static/style/style.css]: I believe it isn't redundant since we removed the margin-bottom from the now non-existent testplan-img class.

File: theme/static/style/style.css (hunk 40 above)
Comment (koksyncho) [reply theme/static/style/style.css]: box-sizing: border-box doesn't work for me, the images go out of the screen due to the margin, that is why I left the margin at 1.8rem and lowered the width to 95%. The same goes for the comment below this one.

**Comment by koksyncho** [comment]
Would you like the responsive part of the style.css to be taken out completely and committed separately or, as you said in one of the latest comments, to comment all of it and start uncommenting the elements one by one as they are remade?

File: theme/static/style/style.css (hunk 41 above)
Comment (koksyncho) [reply theme/static/style/style.css]: box-sizing: border-box doesn't work for me, the images go out of the screen due to the margin, that is why I left the margin at 1.8rem and lowered the width to 95%.

**Comment by atodorov** [comment]
@koksyncho please remove the overriding of styles for responsive completely and let's first merge the white-space changes and the removal of inline styles.

**Comment by koksyncho** [comment]
@atodorov the responsive styles on the features page have been removed. I will start writing them one by one again in order to evade running into the old overriding problems.

File: theme/static/style/style.css [hunk 43]
```diff
@@ -400,11 +426,15 @@ pre code {
...
-    .screenshots .img-index {
+    .screenshots .col-2, .img-index {
         display: none;
         width: 0;
     }
 
+    .screenshots .col-8 {
+        width: 100%;
+    }
+
```
Comment (atodorov) [inline theme/static/style/style.css]: Move this back to the top from where you have deleted it. It controls the styling of the screenshots image in the main page and for screens under 1360px we still want to have the image span the full width. With the current change the images goes on to the left which is a regression from existing behavior.

**Review by atodorov — CHANGES_REQUESTED** [review]
Revert the `.screenshots .col-8` change and this is ready to go.
